service/service.go

```diff
@@ -35,9 +38,19 @@ type repository interface {
 		filter models.FilterComments, sort models.SortComments) ([]*models.Comment, error)
 }
 
-func NewService(r repository, c *configs.Config) *Service {
-	return &Service{
-		repo:   r,
-		config: c,
+func NewService(r repository, privateKeyFile string) (*Service, error) {
+	privateKey, err := ioutil.ReadFile(privateKeyFile)
```

Comment:
please read private key outside of service.... does service is responsible to read files?